### PR TITLE
APK Release configuration for ANC

### LINKED ISSUE #1152: Automate APK releases for ANC (CI/CD)
In addition to the automated APK release set up we have for Quest, we need to have a separate set up for ANC. 

ANC will be in future integrated in quest. A semi automated approach for this release is preferred where the release can be triggered manually and on demand. All other aspects of the release like building and signing the APK will still be automated by CI.

### PR DESCRIPTION
**Pull request by ndegwamartin**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #1152

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/build.gradle [hunk 1]
```diff
@@ -44,7 +44,7 @@ android {
     signingConfigs {
         release {
 
-            v1SigningEnabled true
+            v1SigningEnabled false
```
Comment (f-odhiambo) [inline android/quest/build.gradle:47]: Why is signing false? Will this be only true when we want to publish to the playstore?

File: .github/workflows/apk-release.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This workflow will build a Java project with Gradle
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
 
-name: APK Release
+name: Quest APK Release
```
Comment (f-odhiambo) [inline .github/workflows/apk-release.yml:4]: What happens to a non Quest APK release ? in the case of Malawi core?

**Review by f-odhiambo — APPROVED** [review]
Added a few comments , otherwise LGTM

File: .github/workflows/apk-release.yml (hunk 2 above)
Comment (ndegwamartin) [reply .github/workflows/apk-release.yml:4]: Depends if we expect that as a recurring pattern i.e. creating new modules outside of Quest. 

We can implement a solution once we add the Malawi core module

File: android/quest/build.gradle (hunk 1 above)
Comment (ndegwamartin) [reply android/quest/build.gradle:47]: v1 is false because of a security vulnerability on some Android versions. We should be okay with just using v2.

**Pull request merged**